spec/controllers/v1/post911_gi_bill_statuses_controller_spec.rb

```diff
@@ -63,9 +65,14 @@
     end
   end
 
+  # TO-DO: Remove this suite of tests after transition of LTS to 24/7 availability
   context 'outside working hours' do
     # midnight
-    before { Timecop.freeze(tz.parse('2nd Feb 1993 00:00:00')) }
+    before do
+      Flipper.disable(:sob_updated_design)
```

Comment:
The current specs do not test the new code when the sob_updated_design is enabled. Can you please make create specs for  Flipper.enable(:sob_updated_design)?